shopelectro/tests/tests_models.py

```diff
@@ -86,3 +86,32 @@ def test_get_brands_from_products(self):
                 p: p.tags.filter(group__name=settings.BRAND_TAG_GROUP_NAME).first()
                 for p in products
             }
+
+    def test_get_catalog_blocks(self):
+        """Perform four queries to fetch in batch blocks, categories, pages and category's children."""
+        roots = Category.objects.filter(level=0)
+        roots_count = roots.count()
```

Comment:
we can use `roots.count()` directly, without var

Reply:
It'll bring an additional query to `assertNumQueries(4)` scope, that doesn't related to the test